torch/_export/converter.py

```diff
@@ -564,24 +674,35 @@ def __init__(
         self.sample_args = sample_args
         self.sample_kwargs = sample_kwargs
 
-        self.param_names: Set[str] = {name for name, _ in ts_model.named_parameters()}
-        self.buffer_names: Set[str] = {name for name, _ in ts_model.named_buffers()}
+        self.name_to_param_map: Dict[str, torch.Tensor] = (
```

Comment:
nit: what's the case that ts_model is not a jit.ScriptModule? Maybe we could be more explicit about what type of ts_model we can convert e.g. by adding a type annotation for ts_model and instead of `else`, just use one more `if`

Reply:
There is also a `ScriptFunction` if user directly scripts a function instead of an `nn.module`